x-pack/metricbeat/Jenkinsfile.yml

```diff
@@ -0,0 +1,39 @@
+when:
+    branches: true             ## for all the branches
+    changeset:                 ## when PR contains any of those entries in the changeset
+        - "^x-pack/metricbeat/.*"
+        - "@ci"                ## special token regarding the changeset for the ci
+        - "@xpack"             ## special token regarding the changeset for the xpack
+    comments:                  ## when PR comment contains any of those entries
+        - "/test x-pack/metricbeat"
+    labels:                    ## when PR labels matches any of those entries
+        - "x-pack-metricbeat"
+    parameters:                ## when parameter was selected in the UI.
+        - "x-pack-metricbeat"
+    tags: true                 ## for all the tags
+platform: "linux && ubuntu-18" ## default label for all the stages
+stages:
+    build:
+        cloud: "mage build test"
+        withModule: true       ## run the ITs only if the changeset affects a specific module.
+        dirs:                  ## run the cloud tests for the given modules.
+            - "x-pack/metricbeat/module/aws"
+        when:                  ## Aggregate when with the top-level one.
+            parameters:
+                - "awsCloudTests"
+                - "allCloudTests"
```

Comment:
Will linux tests be run if cloud tests are disabled?

Reply:
See https://github.com/elastic/beats/pull/20104/commits/987d8802ac26edf4be6f5755224e6118b23aa018